src/SFA.DAS.AssessorService.Application/Handlers/Staff/StaffSearchHandler.cs

```diff
@@ -107,9 +107,9 @@ private async Task<StaffReposSearchResult> Search(StaffSearchRequest request)
                 var sr = new StaffReposSearchResult
```

Comment:
This code works, albeit is a bit smelly. Just wondering if this looks cleaner:

var results = await _staffIlrRepository.SearchForLearnerByCertificateReference(request.SearchQuery);

return new StaffReposSearchResult
{
	DisplayEpao = true,
        PageOfResults = results,
	TotalCount = results.Count()
};